internal/confmapprovider/discovery/bundle/bundle_other.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !windows
```

Comment:
Can you explain the split on Windows? Are some bundle rules not compatible with Windows and therefore you're excluding them from the build for Windows?

Reply:
native collectd monitors like mysql aren't available on windows: https://github.com/signalfx/splunk-otel-collector/blob/main/pkg/receiver/smartagentreceiver/config.go#L39. Moving forward we'll need similar platform-specific bundles for other ones (like darwin).

Reply:
the specifics of ^ are undefined for best experience and are currently based on what we provide agent bundles for.